README.md

```diff
@@ -1 +1,156 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Enumerable] <a name="about-project"></a>
+
+**[Enumerable]** is a pair programming project where you sort, and organize different values.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Sort]**
+- **[Sort Max value, Sort Min value]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Install Ruby with RubyInstaller, then open your CMD and type 'irb'
+
+Now you can start using Ruby!
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Have installed Ruby
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd desire-folder
+  git clone https://github.com/Mike47ip/morse-code.git
+
+### Install
+
+No installation is needed.
+
+### Usage
+
+To run the project, execute the following command:
+
+No command existing until now.
+
+### Run tests
+
+To run tests, run the following command:
+
+No test have been created yet for this project.
+
+### Deployment
+
+You can deploy this project using:
+
+No deployment have made yet.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Michael Darkwah**
+
+- GitHub: [@Mike47ip](https://github.com/Mike47ip)
+- Twitter: [@ip_mikee](https://twitter.com/ip_mikee)
+- LinkedIn: [Michael Darkwah](https://linkedin.com/in/michael-darkwah-81
+
+👤 **Agustin Soria**
+
+- GitHub: [@Agustin-Soria](https://github.com/SaveryIV)
+- LinkedIn: [Agustin Soria](https://www.linkedin.com/in/agust%C3%ADn-ricardo-soria-meza-979747228/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Terminal App
+- Optimizing query time
+- Create database diagram
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Mike47ip/morse-code/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ <a name="faq"></a>
+
+N/A
```

Comment:
[OPTIONAL]
- [ ] This is an optional section and since it is unavailable, you can remove it, ensure to also remove it from the headers above your readme file :+1: :sparkler: